nifi-commons/nifi-record/src/main/java/org/apache/nifi/serialization/record/util/FractionalSecondsUtils.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.nifi.serialization.record.util;
+
+import java.time.Instant;
+
+public class FractionalSecondsUtils {
+    private static final long YEAR_TEN_THOUSAND = 253_402_300_800_000L;
+
+    private FractionalSecondsUtils() {
+    }
+
+    public static Instant tryParseAsNumber(final String value) {
```

Comment:
Recommend renaming this method to `toInstant()` as well, following the pattern of the other methods.